src/pages/BookmarksPage.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import DiveWish from '../DiveWish.js';
+
+export default function BookmarksPage({
+  bookmarkedWishes,
+  onToggleBookmark,
+  onDeleteDiveWish,
+}) {
+  return (
+    <>
+      <header>
+        <h1>Your favorites</h1>
+      </header>
+      <main>
+        <Wrapper>
+          {bookmarkedWishes.length > 0 ? (
+            bookmarkedWishes.map(wish => (
+              <DiveWish
+                key={wish.id}
+                id={wish.id}
+                destination={wish.destination}
+                notes={wish.notes}
+                isBookmarked={wish.isBookmarked}
+                onToggleBookmark={() => onToggleBookmark(wish.id)}
+                onDeleteDiveWish={onDeleteDiveWish}
```

Comment:
Ah! You could do the same here with `onDeleteDiveWish` then you don't have to pass the `id`.
```suggestion
                onToggleBookmark={() => onToggleBookmark(wish.id)}
                onDeleteDiveWish={() => onDeleteDiveWish(wish.id)}
```
And it would be a bit more consistent with what you already wrote.

Reply:
True, thx!